src/constructs/numbers.md

```diff
@@ -2,6 +2,120 @@
 title: Numbers
 ---
 
-## On the Subject of Numbers
+Grain has a unified number type, called `Number`, which contains integer (whole) numbers, floating-point (decimal) numbers, and rational (fractional) numbers. Grain also has separate integer and float types for programs which require more precise typechecking.
 
-This documentation hasn't yet been migrated here (or written, to be honest). In the meanwhile, please check out the [Grain compiler README](https://github.com/grain-lang/grain/blob/master/README.md) for more information. Consider contributing by fleshing out this page yourself!
\ No newline at end of file
+```grain
+42 // an integer
+```
+
+```grain
+1.23 // a floating-point number
+```
+
+```grain
+1/3 // a rational number
+```
```

Comment:
Move these into their section, because it sounds weird to start the sections with "Also" or "In addition"